Gems/Atom/Asset/Shader/Code/Source/Platform/Linux/platform_builders_linux.cmake

```diff
@@ -4,3 +4,74 @@
 # SPDX-License-Identifier: Apache-2.0 OR MIT
 #
 #
+
+# Shader Headers
+#ly_add_target_files(
```

Comment:
The next few lines are not commented out in the Mac version, so we can probably keep them on Linux too?

Reply:
They should be un-commented as we want linux to build dx12 shaders. Ideally I would wait to test this properly with azslc/dxc before merging the shader changes but I am not opposed to merging them in early as no one is really using Linux and azslc package is coming soon